images/rabbitmq/definitions.json

```diff
@@ -0,0 +1,14 @@
+{
+  "vhosts":[
+        {"name": "${RABBITMQ_DEFAULT_VHOST}"}
+  ],
+  "users": [
+        { "name": "${RABBITMQ_DEFAULT_USER}", "password": "${RABBITMQ_DEFAULT_PASS}", "tags": "administrator" }
+  ],
+  "permissions":[
+        { "user": "${RABBITMQ_DEFAULT_USER}", "vhost": "${RABBITMQ_DEFAULT_VHOST}", "configure": ".*", "write": ".*", "read": ".*" }
+  ],
+  "policies":[
+        {"vhost":"${RABBITMQ_DEFAULT_VHOST}","name":"lagoon-ha","pattern":"${RABBITMQ_DEFAULT_HA_PATTERN}", "definition":{"ha-mode":"exactly","ha-params":2,"ha-sync-mode":"automatic","ha-sync-batch-size":5}}
```

Comment:
reading https://www.rabbitmq.com/ha.html it seems that we should set `ha-sync-batch-size` to `1` as we want every single message by synced and not packs of 5.

Reply:
In the link you sent I see: 

> Earlier versions will synchronise 1 message at a time by default. By synchronising messages in batches, the synchronisation process can be sped up considerably.

It seems that a batch-size greater than 1, increase the speed of synchronization process.